backend/foodbank_southlondon/bff/parsers.py

```diff
@@ -24,5 +24,5 @@
 summary_params.add_argument(_requests_params_args["event_names"])
 
 action_params = reqparse.RequestParser(trim=True)
-action_params.add_argument("ignore_warnings", type=inputs.boolean, required=False, default=True, help="Whether warnings should be ignored or not. If "
+action_params.add_argument("ignore_warnings", type=inputs.boolean, required=False, default=False, help="Whether warnings should be ignored or not. If "
```

Comment:
The frontend always sends this value now but it still seems like this should be the default?